### PR TITLE
feat: toggle off repo sync completely

### PR DESCRIPTION
**Pull request by jthegedus**
# Summary

This builds on #957 by allowing the plugin shortname repo sync to be toggled off completely with:

```
# $HOME/.asdfrc
plugin_repository_last_check_duration = never
```

Fixes: #978 

## Other Information

<details>
<summary>Docs changes comparison</summary>

image

</details>

### CODE REVIEW

**Comment by jthegedus** [comment]
I am still not happy with the docs for these. I will propose another change tomorrow.

**Comment by Stratus3D** [comment]
Sorry for the late review. Looking at this now.

File: docs/manage/configuration.md [hunk 1]
```diff
@@ -41,18 +42,51 @@ Edit the file directly or use `asdf local` (or `asdf global`) which updates it.
...
+| `yes`                                                      | Release Candidate is used |
+
+### `always_keep_download`
+
+Configure the `asdf install` command to keep or delete the source code or binary it downloads.
+
+| Options                                                    | Description                                |
+| :--------------------------------------------------------- | :----------------------------------------- |
+| `no` <Badge type="tip" text="default" vertical="middle" /> | Delete source code or binary after install |
+| `yes`                                                      | Keep source code or binary after install   |
```
Comment (Stratus3D) [inline docs/manage/configuration.md:79]: Tables seem a little unnecessary for some of these options.

Also, this:

> Delete source code or binary after install

Should be:

> Delete source code or binary after successful install

**Review by Stratus3D — APPROVED** [review]

File: docs/manage/configuration.md (hunk 1 above)
Comment (jthegedus) [reply docs/manage/configuration.md:79]: >Tables seem a little unnecessary for some of these options.

I found the the `plugin_repository_last_check_duration` very wordy and unclear without the table. After adding one it seemed best to be consistent, though I agree the binary settings don't benefit all that much.

**Pull request merged**